res/controllers/Denon-DN-S3700.qml

```diff
@@ -0,0 +1,17 @@
+import QtQuick
```

Comment:
we don't actually want QtQuick (since thats the GUI framework). You instead just want the language basics `QtQml`. You may also add a version (not sure whats appropriate, in the preexisting qml files we have in mixxx we use 2.12, but thats quite old (I think from Qt 5.12 still)).
```suggestion
import QtQml
```